### DISCUSSION TITLE
Missing Assembly and Shimmer

### DISCUSSION THREAD

**Discussion by bilIvoo** [iotaledger/firefly#7566] [Help & Support]
Hello,
my shimmer and assembly are missing. I did all cycles of staking for shimmer and assemly in Firefly wallet and than with 2.0 update is all my shimmer and assemby are gone.But funny enough i have 4 Iotas more on my iota firefly wallet.I have installed the firefly for shimmer but i cannot acces my shimer. I had only one firefly wallet for all 3 (iota,shimmer,assembly). My iota on iota wallet is there but shimmer and assemby are gone. Anyone else with this problem? What do i need to do? Any advice is helpful. Ivo

**Comment by begonaalvarezd** [comment]
Hi, please download the latest Firefly IOTA version 2.0.2. The network was recently upgraded so the old firefly doesnt work anymore

- Assembly: please read https://blog.iota.org/stardust-upgrade-iota-tokenomics/. The assembly project does not exist anymore, but your rewards got airdropped to you in form of IOTAs. They will appear in the new version 2.0.2 in the vesting tab. If you are missing balances you can run the balance finder from settings > advanced > balance finder, or if you are missing rewards you can also run the rewards finder from the vesting tab, the '....' menu, search vesting rewards
- Shimmer: what the old firefly version displayed was just a representation of what you owned, but they were not the real shimmers. You must download the firefly shimmer app to control those funds, you can read more about it here https://blog.shimmer.network/how-to-claim-smr-firefly/

Will close the issue because the new firefly app is not intended to display old staking rewards, but the new iota airdrop is on the new app and the shimmer tokens should be managed from the shimmer app. 

Please let us know if you have any other issues

**Comment by bilIvoo** [comment]
Thank you! It worked! Been in IOTA project since 2017. This helped a lot. Have a nice day!
